Added NBAGrid to the dles.json

monsdar opened the pull request:
NBAGrid is a game similar to Geogrid where the user needs to guess NBA players that match a number of categories.

aukspot commented:
Hi! I appreciate you taking the initiative to add NBAGrid to the DLES, and I'm sorry it's taken me so long to respond. The minimal UI is nice, but I feel like the game could use some instructions and a way to go to a specific puzzle without needing to repeatedly press the arrows. But the main thing is the lack of instructions!

monsdar commented:
Thanks for the feedback! You're right in the missing instructions. I added a dialog to the game that summarizes how it's being played.

I have mixed feelings on the game-selection though. I think the UX could be clunky. What fits better (and covers the usecase of "Let me play more") would be a "Jump to a game I haven't played yet"-button. I need to find a good name, but I think you get what I mean. What do you think?

aukspot commented:
Nice! The game seems to only feature active NBA players, maybe the instructions could say that somewhere.

A button to play a random unplayed puzzle would be fun. For a game selection, you could try a dropdown monthly calendar

monsdar commented:
Good point, I added that only active players are guessable.

I also added a button that goes to the next unplayed grid. It is shown after the user has finished a game (along with the Stats and About buttons). That way after completing a game the user can directly go to the next one he hasn't finished yet.

I'm not really sure about the calendar. I'd say the usecase to pick a specific date is not very common - but adding a calendar only for this would make the code and page a lot more complex. Same for UX - how do you display a calendar on mobile in a good way? What needs to go along the calendar - i.e. visual cues for completed and incomplete days? What about score, or rank for a specific date?

I like the way Geogrid does it, in displaying a list of past games along with the score of the user. But as it does not add that much to the game I'd leave it out for the time being. Perhaps it's a good addition in the future, but not for now.

aukspot commented:
That's fair about the calendar! I have one for my game ([https://estimate-me.aukspot.com/](https://estimate-me.aukspot.com/), when you click the date at the top) and it is a bit of extra work. 

I think this is good to add to the list now! We'll need to update the entry to exclude the tags since I deprecated them. Then it should automatically add an id to it when the Github Actions run. (fingers crossed)

monsdar commented:
Cool. I synced with latest main and updated the dles-entry
